static/app/components/events/contexts/utils.tsx

```diff
@@ -27,102 +26,37 @@ import type {AvatarUser} from 'sentry/types/user';
+import {getDefaultContextData} from './default';
+import {getKnownDeviceContextData, getUnknownDeviceContextData} from './device';
+import {getKnownGpuContextData, getUnknownGpuContextData} from './gpu';
 import {
   getKnownMemoryInfoContextData,
   getUnknownMemoryInfoContextData,
-  MemoryInfoEventContext,
 } from './memoryInfo';
 import {
   getKnownOperatingSystemContextData,
   getUnknownOperatingSystemContextData,
-  OperatingSystemEventContext,
 } from './operatingSystem';
 import {
   getKnownPlatformContextData,
   getPlatformContextIcon,
   getUnknownPlatformContextData,
   KNOWN_PLATFORM_CONTEXTS,
 } from './platform';
-import {
-  getKnownProfileContextData,
-  getUnknownProfileContextData,
-  ProfileEventContext,
-} from './profile';
-import {getReduxContextData, ReduxContext} from './redux';
-import {
-  getKnownReplayContextData,
-  getUnknownReplayContextData,
-  ReplayEventContext,
-} from './replay';
-import {
-  getKnownRuntimeContextData,
-  getUnknownRuntimeContextData,
-  RuntimeEventContext,
-} from './runtime';
-import {
-  getKnownStateContextData,
-  getUnknownStateContextData,
-  StateEventContext,
-} from './state';
+import {getKnownProfileContextData, getUnknownProfileContextData} from './profile';
+import {getReduxContextData} from './redux';
+import {getKnownReplayContextData, getUnknownReplayContextData} from './replay';
+import {getKnownRuntimeContextData, getUnknownRuntimeContextData} from './runtime';
+import {getKnownStateContextData, getUnknownStateContextData} from './state';
 import {
   getKnownThreadPoolInfoContextData,
   getUnknownThreadPoolInfoContextData,
-  ThreadPoolInfoEventContext,
 } from './threadPoolInfo';
-import {
-  getKnownTraceContextData,
-  getUnknownTraceContextData,
-  TraceEventContext,
-} from './trace';
-import {
-  getKnownUnityContextData,
-  getUnknownUnityContextData,
-  UnityEventContext,
-} from './unity';
-import {
-  getKnownUserContextData,
-  getUnknownUserContextData,
-  UserEventContext,
-} from './user';
-
-const CONTEXT_TYPES = {
-  default: DefaultContext,
-  app: AppEventContext,
-  device: DeviceEventContext,
-  memory_info: MemoryInfoEventContext,
-  browser: BrowserEventContext,
-  os: OperatingSystemEventContext,
-  unity: UnityEventContext,
-  runtime: RuntimeEventContext,
-  user: UserEventContext,
-  gpu: GPUEventContext,
-  trace: TraceEventContext,
-  threadpool_info: ThreadPoolInfoEventContext,
-  state: StateEventContext,
-  profile: ProfileEventContext,
-  replay: ReplayEventContext,
-  // 'redux.state' will be replaced with more generic context called 'state'
-  'redux.state': ReduxContext,
-  // 'ThreadPool Info' will be replaced with 'threadpool_info' but
-  // we want to keep it here for now so it works for existing versions
-  'ThreadPool Info': ThreadPoolInfoEventContext,
-  // 'Memory Info' will be replaced with 'memory_info' but
-  // we want to keep it here for now so it works for existing versions
-  'Memory Info': MemoryInfoEventContext,
-};
```

Comment:
So literally all of these components can be removed, but each will be doing a change for each will be ~200 lines, so I will do those in follow up PRs